css/main.css

```diff
@@ -208,5 +405,119 @@ img.logo {
   img.logo {
     width: 5em;
     height: 5em; } }
+.search-results {
+  display: flex;
+  flex-direction: row;
```

Comment:
There is a mixin defined for these two properties ` display: flex;
  flex-direction: row;`. I think for consistency you can use it `@include raw`